packages/shared/src/lib/featureValues.ts

```diff
@@ -28,11 +28,6 @@ export enum UserAcquisition {
   V1 = 'v1',
 }
 
-export enum FeedAdSpot {
-  Control = defaultFeedContextData.adSpot,
```

Comment:
warning  'defaultFeedContextData' is defined but never used  @typescript-eslint/no-unused-vars